ProjectManager/ProjectManager/Common/Database/RealmDatabaseManager.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  RealmDatabaseManager.swift
+//  ProjectManager
+//
+//  Created by seohyeon park on 2022/09/12.
+//
+
+import RealmSwift
+
+class RealmDatabaseManager: DatabaseManager {
+    let realm = try? Realm()
+
+    func createDatabase(data: Object) {
+        guard let realm = realm,
+              ((try? realm.write({ realm.add(data) })) != nil) else {
+            return
+        }
+    }
+
+    func readDatabase() -> [TaskModel] {
+        guard let realm = realm else {
+            return [TaskModel]()
+        }
+
+        var database = [TaskModel]()
```

Comment:
```suggestion
        var taskModels = [TaskModel]()
```

taskModels라고 하는게 더 나아보이네요 :) 
데이터베이스는 아니니까요!

Reply:
네이밍 수정 했습니다😄